pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,542 @@
+      rel="stylesheet"
+      type="text/css"
+    />
+    <style>
+      header {
+        height: 8vh;
+        color: #545454;
+        background-color: #DDB556;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      amp-story-player {
+        margin: 1rem auto;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>
+        Page Header
+      </h1>
+    </header>
+    <h1>
+      Article Title
+    </h1>
+    <amp-story-player style="width: 370px; height: 622px;">
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s1"></a>
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s2"></a>
+      <a href="https://ampfest-story-player-demo.web.app/examples/amp-story/AMPFestPlayerDemo/s3"></a>
+    </amp-story-player>
+  </body>
+</html>
+```
+[/example]
+
+
+The Web Story Player displays the first embedded story. It automatically presents the user with the next story’s cover page after finishing the previous, or using the swiping motion to skip. The Web Story Player does not have any built-in UI functionality to inform the user there are multiple stories embedded in a single player. Instead, build [user entry points](#user-entry-points) to display available stories and allow user selection.  
+
+
+### Circular story wrapping
+
+You can create a circular consumption of a set of stories by adding the `circular-wrapping` action to the JSON configuration. When `circular-wrapping` is included, the Web Story Player displays the first story to users after they finish the last story.
+
+```html
+<amp-story-player>
+ <script type="application/json">
+   {
+     "behavior": {
+       "on": "end",
+       "action": "circular-wrapping"
+     }
+   }
+ </script>
+ <a href="./story1.html"> ... </a>
+ <a href="./story2.html"> ... </a>
+  ...
+</amp-story-player>
+``` 
+
+The JSON configuration must be a direct child of the `<amp-story-player>` element and include the `type="application/json" attribute. 
+
+
+### Infinite stories
+
+You can create an "infinite scroll" experience by fetching more stories as the user navigates through them in the Web Story Player. To do so, include the `fetch` action and specify an endpoint in the JSON configuration. The player automatically fetches more stories as the user gets closer to the last loaded story. 
+
+```html
+<amp-story-player>
+ <script type="application/json">
+   {
+     "behavior": {
+       "on": "end",
+       "action": "fetch",
+       "endpoint": "https://example.com/my-endpoint.json?offset=${offset}"
+     }
+   }
+ </script>
+ <a href="./story1.html"> ... </a>
+ <a href="./story2.html"> ... </a>
+  ...
+```
+
+The JSON configuration must be a direct child of the `<amp-story-player>` element and include the `type="application/json" attribute. For pagination, use the optional `endpoint` url variable `${offset}`  to add a parameter.
```

Comment:
```suggestion
The JSON configuration must be a direct child of the `<amp-story-player>` element and include the `type="application/json"` attribute. For pagination, use the optional `endpoint` url variable `${offset}`  to add a parameter.
```

Reply:
I think we should specify that the Player code will replace ${offset} with the actual offset.
Eg: if the publisher provides `https://example.com/my-endpoint.json?offset=${offset}` and the Player has 5 stories loaded, we'd send `https://example.com/my-endpoint.json?offset=5`